templates/widget.html

```diff
@@ -1,30 +1,34 @@
 <html>
     <head>
-        <link
-            rel="stylesheet"
-            href="https://aspencdn.me/bootstrap/4.4.1/css/bootstrap.min.css"
-        />
-        <link rel="stylesheet" id="iframe-theme" href="/static/iframe-{{ theme }}.css" />
+        <link rel="stylesheet" href="https://aspencdn.me/bootstrap/4.4.1/css/bootstrap.min.css"/>
+        <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons">
+        <link rel="stylesheet" href="https://code.getmdl.io/1.3.0/material.{{ primary }}-{{ secondary }}.min.css">
+        <script defer src="https://code.getmdl.io/1.3.0/material.min.js"></script>
     </head>
     <body style="overflow-x: hidden;">
         <div class="row row-cols-1 row-cols-md-3">
             {% for repo in repos %}
             <div class="col mb-4">
-                <div class="card">
-                    <h5 class="card-header">
-                        {{repo["name"]}}
-                    </h5>
-                    <div class="card-body">
+                <div class="mdl-card mdl-shadow--4dp " style="width: 100%; margin: 1em 1em; border-radius: 0.5em">
```

Comment:
I will need to change some things here because if we do this it will only work with the material theme.